server/index-node/src/schema.graphql

```diff
@@ -4,7 +4,16 @@ scalar Bytes
 scalar ID
 scalar Int
 scalar String
+
+# An obscure object type. Note that this is equivalent to a JSON object, rather
+# than a generic JSON value; as such it cannot be e.g. a string or an integer.
 scalar JSONObject
+# A string in the format of `YYYY-MM-DD`. This is the standard adopted by
+# RFC3339, which is equivalent to the "ISO 8601 extended format".
+# See also:
+# - https://datatracker.ietf.org/doc/html/rfc3339#section-5.6
+# - https://en.wikipedia.org/wiki/ISO_8601#Calendar_dates
+scalar Date
```

Comment:
I'm unsure about adding a Scalar. We haven't done this in a while, so I don't know if it's fine and we have already discussed about this as a team.

Reply:
Yeah, that's fair. We were unsure about the same when introducing `JSONObject`: https://github.com/graphprotocol/graph-node/pull/3275#discussion_r817147165. We ultimately decided that was fine but didn't really discuss `Date` as far as I can remember! Let's wait for some feedback.

Reply:
Since this is just the indexing status api, I don't mind experimenting with a date scalar.

Reply:
Since this is just the indexing status api, I don't mind experimenting with a date scalar.

Reply:
Marking as resolved. It seems we're okay with this.